src/main.ts

```diff
@@ -52,7 +52,16 @@ async function startup(): Promise<void> {
 
   try {
     // Load the settings file.
-    settingsFilePath = Settings.loadConfiguration(["/run/secrets/settings", "/config/settings.json"]);
+    settingsFilePath = Settings.loadConfiguration([
+      {
+        settingsFilePath: "/run/secrets/settings",
+        secretsFilePath: "/run/secrets/secrets",
+      },
+      {
+        settingsFilePath: "/config/settings.json",
+        secretsFilePath: "/config/secrets.json",
+      },
+    ]);
```

Comment:
@danecreekphotography Also, any tests for this?

Reply:
Same comment as above. Add tests where you believe they will add value. Skip them otherwise.

Reply:
Sounds good